packages/core/src/core/adapters/adapters.ts

```diff
@@ -46,6 +49,7 @@ export const initAdapters = (
       if (!creator) {
         throw new Error(`${adapter} adapter is not registered.`)
       }
+      log.debug('Using the following config for %s adapter: %o', adapter, config[adapter]?.config?.value)
```

Comment:
the config that is passed to the adapter is the one from context, there is no point in going back to the `config` object that is being iterated by `_.mapValues`
```suggestion
      log.debug('Using the following config for %s adapter: %o', adapter, context.config?.value)
```

also, I think the `%o` won't recurse into complex configuration values (default depth is 2), not sure if that is a bad thing, just want to make sure it is not a problem with what you are trying to achieve

Reply:
This is the print we got for SF, I don't see any depth issue here:
```2020-09-05T09:08:33.695Z debug core/core/adapters/adapters using the following config for salesforce adapter: {
  metadataTypesSkippedList: [
    'Report',
    'ReportType',
    'ReportFolder',
    'Dashboard',
    'DashboardFolder',
    'Profile',
    [length]: 6
  ],
  instancesRegexSkippedList: [ '^EmailTemplate.MarketoEmailTemplates', [length]: 1 ],
  maxConcurrentRetrieveRequests: 3,
  maxItemsInRetrieveRequest: 2500,
  enableHideTypesInNacls: false,
  dataManagement: [
    {
      name: 'CPQ',
      enabled: false,
      isNameBasedID: true,
      includeNamespaces: [ 'SBQQ', [length]: 1 ],
      includeObjects: [ 'Product2', 'Pricebook2', 'PricebookEntry', [length]: 3 ],
      excludeObjects: [
        'SBQQ__ContractedPrice__c',
        'SBQQ__Quote__c',
        'SBQQ__QuoteLine__c',
        'SBQQ__Subscription__c',
        'SBQQ__SubscribedAsset__c',
        'SBQQ__SubscribedQuoteLine__c',
        'SBQQ__SubscriptionConsumptionRate__c',
        'SBQQ__SubscriptionConsumptionSchedule__c',
        'SBQQ__WebQuote__c',
        'SBQQ__WebQuoteLine__c',
        [length]: 10
      ]
    },
    [length]: 1
  ]
}